pkg/netdevice/netInfoProviders.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 /*
-   rdmaInfoProvider provides the RDMA information
+rdmaInfoProvider provides the RDMA information
```

Comment:
indentation changes are not related to this PR. can you either:

1. move them to separate PR
2. move them to a separate commit in this PR

Reply:
Removed indentation changes from this PR